### DISCUSSION TITLE
Fatal Error in Python.

### DISCUSSION THREAD

**Discussion by AbdallahAdham** [community/community#21394] [Other Feature Feedback, Questions, & Ideas]
<p>I’m facing this problem for about 4 months I’ve searched everywhere but no solution.</p>
<p>Fatal Python error: initfsencoding: unable to load the file system codec<br>
ModuleNotFoundError: No module named ‘encodings’</p>
<p>Current thread 0x00004e04 (most recent call first):</p>
<p>my Os is Windows 10</p>
<p>please give me any solution when I can’t code anything it gives me that error. it happened after I installed another python 2 and I’m using python 3.</p>
<p>Thanks very much.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/dragontiger321">@dragontiger321</a>,</p>
<p>Thanks for being here! Have you tried to <em>a__dd Python to environment variables</em> in Advanced Options?</p>

**Comment by AbdallahAdham** [comment]
<p>Yes, I’ve tried many times but still, this error is happening, and now there is no interpreter, it says it is missing.</p>
<p>Do I have to recover or backup my computer, I’m puzzled and can’t find any solution.</p>
<p>Thanks for your help.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>I googled it and found a possibly related, now resolved <a href="https://bugs.python.org/issue35596" rel="nofollow noopener">python bug</a>.</p>
<p>If you can, upgrade your Python 3 to the absolute latest, or downgrade to a working 3.x. It is also possible your Python 2 may be interfering: you might try de-installing both, then install _one_ of 2.x or 3.x (but not both).</p>
